app/service/rest_svc.py

```diff
@@ -124,8 +125,8 @@ async def display_result(self, data):
                 content = self.get_service('file_svc').read_result_file('%s' % link_id)
                 return dict(link=link.display, output=content)
             except FileNotFoundError:
-                return ''
-        return ''
+                return dict(link=link.display, output='')
+        return dict()
```

Comment:
Is there a reason we don't use dictionary literals very much in Caldera? I see `dict` all over the place, but rarely `{}`.

Reply:
It's a styling thing that was inherited from a former project head. We've just kind of stuck with it to keep things as homogenous as possible.